webpack.config.js

```diff
@@ -0,0 +1,81 @@
+var webpack = require('webpack');
+var path = require('path');
+
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+module.exports = {
+
+    entry: {
+        app: './src/main.ts',
+        vendor: './src/vendor.ts',
+        polyfills: './src/polyfills.ts'
+    },
+
+    output: {
+        path: path.join(__dirname, 'dist', 'js'),
+        filename: '[name].js'
+    },
+
+    resolve: {
+        extensions: ['.ts', '.js', '.json'],
+        modules: ["node_modules", "bower_components"],
+        descriptionFiles: ["package.json", "bower.json"]
+    },
+    
+    module: {
+        rules: [{
+            test: /\.ts$/,
+            use: ['awesome-typescript-loader', 'angular2-template-loader']
+        }, {
+            test: /\.html$/,
+            use: "html-loader"
+        }, {
+            test: /\.(png|jpe?g|gif|svg|woff|woff2|ttf|eot|ico|otf)$/,
+            use: 'file-loader?name=assets/[name].[ext]'
+        }, {
+            test: /\.less$/,
+            use: ['raw-loader', 'less-loader']
```

Comment:
We should run less files through the css-loader as well to process any images, fonts etc.. We will likely also need to include the to-string-loader or css-to-string-loader.